src/js/components/RadioButtonGroup/stories/CustomTheme.js

```diff
@@ -6,6 +6,12 @@ import { grommet } from 'grommet/themes';
 import { deepMerge } from 'grommet/utils';
 
 const customTheme = deepMerge(grommet, {
+  radioButtonGroup: {
+    container: {
+      gap: 'xlarge',
+      animation: 'fadeIn',
```

Comment:
Due to this animation chromatic isn't picking up the changes.
Can you disable chromatic for this file? https://www.chromatic.com/snapshot?appId=5d9774839a6eff00203f5cbf&id=5f2c55d4f321ca0022245fa1

Reply:
gotcha